### DISCUSSION TITLE
Debug mode options explained

### DISCUSSION THREAD

**Discussion by Ingood5** [d4rken-org/sdmaid-se#534] [Q&A] [answered]
Что означают все эти пункты?
![IMG_20230404_222301](https://user-images.githubusercontent.com/60130882/229899400-a00101ab-17ff-42e4-9e35-28c7e62d6491.jpg)

**Comment by d4rken** [comment, marked as answer]
> Enable trace mode

This turn on a lot of extra log output, very detailed information about scanned files and every resource SD Maid requires. It helps to troubleshoot some bugs, but it also makes SD Maid slower because of so much logging.

> Enable dry-run mode

Disables deletion, so it's only faked. Makes some things easier to test if I don't have to create test data every time.

> Reload data areas

Before running operations, SD Maid determines available data areas (you can see them in settings). They are automatically reloaded when permissions change, or when this button is pressed :).

> Reload apps

SD Maid has an internal cache of applications that is normally updated only when apps are installed or removed. This button clears it and reloads it.

> Checks root access

Just checks root access, without any other complicated operations.

**Comment by Ingood5** [comment]
Информация, собранная с помощью "Enable trace mode" отправляется в автоматическом отчёте?

**Comment by d4rken** [comment]
> Информация, собранная с помощью "Enable trace mode" отправляется в автоматическом отчёте?

No

Trace-mode log output is generally "Priority VERBOSE", and the automatic reports only include "Priority DEBUG" (and higher).

**Comment by Ingood5** [comment]
Тогда где используется информация, полученная в режиме трассировки? 🤔 В debug log? Не понял каким образом она доставляется вам.

**Comment by d4rken** [comment]
[In debug logs](https://github.com/d4rken-org/sdmaid-se/wiki/Bugs#debug-log), but it's only necessary if I ask for it.
